docs/user-guide/102-configuration.md

```diff
@@ -176,8 +175,9 @@ status:
 
 The output contains the `desiredState` applied by the Policy for the given Node.
 It also contains a list of conditions. This list is more detailed than the one
-in Policy. It shows whether the Policy matched given Node
-(`AllSelectorsMatching`), if the `desiredState` is currently being applied on
+in Policy. It shows whether the max unavailable node limits has reached so the
```

Comment:
Maybe we could reformulate this s little, wdyt?
`It shows whether the max unavailable node limits has reached` -> `It shows whether the limit of unavailable nods was reached so the node has to wait for other nodes to finish applying network configuration ('Pending')`

Reply:
Done